### DISCUSSION TITLE
Can I change one class to another class in the labeled images?

### DISCUSSION THREAD

**Discussion by udkii** [wkentaro/labelme#1068] [Q&A / Help] [answered]
I am doing instance segmentation using labelme. The output file comes as a 'jpg' and a 'json' file. The json file contains the coordinate values of each polygon and their class name.

I've been labeling with a total of 8 classes so far. Completed about 400 images in total.
By the way, I want to reduce the number of classes. ( I wnat to change 8 classes to 5)
So I want to change class A, B to class C.

Is there any other way other than manually changing the labels of the images by hand? I am wondering if there is a way to change class A to class C at once using python code.

Thanks you so much for reading.

**Comment by wkentaro** [comment, marked as answer]
Well, you can write Python code for that. It's a very simple script, right?

Or you can probably do this: (be careful, and better to backup the folder before trying this)
```
ls *.json | xargs -n1 sed -i 's/"dog"/"person"/g'
```
